src/libs/ajax.js

```diff
@@ -236,6 +236,21 @@ const User = signal => ({
     }
   },
 
+  getSamTosAccepted: async () => {
+    try {
+      const res = await fetchSam('register/user/v1/termsofservice/status', _.merge(authOpts(), { signal }))
+      return await res.json()
```

Comment:
Is `await` necessary both here and on the line above? It seems redundant to me, but maybe it takes time to get the json?

Reply:
It theoretically could take a long time. But it's also just architected to return a promise.